docs/UserGuide.md

```diff
@@ -373,6 +373,30 @@ Examples:
 * `eremovev vid/1 eid/1` 
   * removes the volunteer with id 1 from the event with id 1.
 
+### Undo a Command: `undo`
+
+Undoes an undo-able command
+
+* Undo-able commands include:
+  * vcreate
+  * vdelete
+  * vedit
+  * vclear
+  * ecreate
+  * edelete
+  * eedit
+  * eaddv
+  * eremovev
+  * eaddm
+* Only commands that changed the current history stack can be undone (iVolunteer does not remember commands that were executed in the previous run of the app)
+
+### Redo a Command: `redo`
+
+Redoes the undo command. Only can be executed after an undo command is executed. This command can be thought of as an 'undo' command that can only undo a `undo` command
```

Comment:
AHAHHAHA the last part got me thinking but minor suggestion to put `Only can be executed after an undo command is executed.` under `Restrictions:`

Reply:
missed out on format here too